src/test/scala/org/constellation/Fixtures.scala

```diff
@@ -116,4 +120,28 @@ object Fixtures {
   def makeDummyTransaction(src: String, dst: String, keyPair: KeyPair) =
     TransactionService.createDummyTransaction(src, dst, keyPair)(TransactionChainService[IO]).unsafeRunSync
 
+  def manuallyCreateTransaction(
+                                 src: String = "",
+                                 dst: String = "",
+                                 oeHash: String = "",
+                                 sbHash: String = "",
```

Comment:
shouldn't hashes and signatures be calculated automatically based on the data?

Reply:
I intended this function to be used for creating arbitrary Transaction state for tests. In the most general form these can be simply empty strings.